sdk/core/azure-core/CMakeLists.txt

```diff
@@ -36,6 +36,10 @@ if(BUILD_TRANSPORT_CURL)
   message("Libcurl version ${CURL_VERSION_STRING}")
 endif()
 
+if(BUILD_TRANSPORT_WINHTTP)
+  find_package(wil CONFIG REQUIRED)
```

Comment:
Please also update:
1. `sdk/core/azure-core/cgmanifest.json`
2. `sdk/core/azure-core/vcpkg/Config.cmake.in`